src/client/src/apps/MainRoute/routes/AnalyticsRoute.tsx

```diff
@@ -1,20 +1,41 @@
 import React from 'react'
+import { useSelector } from 'react-redux'
 import { AnalyticsContainer } from '../widgets/analytics'
 import styled from 'styled-components/macro'
+import { externalWindowDefault } from 'rt-platforms'
+import { TearOff } from 'rt-components'
+import { analyticsSelector } from '../layouts'
+import { addLayoutToConfig } from './addLayoutToConfig'
+import { inExternalWindow } from './inExternalWindow'
 
-const AnalyticsRouteStyle = styled.div`
+const AnalyticsRouteStyle = styled.div<{ showOpenFinControls : boolean }>`
   /*height offset is needed for openfin controls*/
-  height: calc(100% - 24px);
-  padding: 0 0.625rem 0rem 0.625rem;
-  overflow-x: scroll;
+  height: ${({ showOpenFinControls }) => (showOpenFinControls ? 'calc(100% - 24px)' : '100%')};
+  padding: 0.5rem 0.625rem 0.25rem 0.625rem;
```

Comment:
I think the approach here is good, but can we rename `showOpenFinControls` to just `useHeightOffset` or something like that?

We try to keep OpenFin-specific references isolated to the OpenFin module as much as possible.

Reply:
done